diff --git a/backend/app/main.py b/backend/app/main.py
index 4fdce3be6b8f47cf75b05d6249b5466e3177a0ef..e2ec704e0b5ee69b527befaf4cabbd755986ffa6 100644
--- a/backend/app/main.py
+++ b/backend/app/main.py
@@ -1,26 +1,27 @@
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
-from app.api import openaq   # asumimos que acá está tu router
+from app.api import openaq as tempo_router
 
 app = FastAPI(title="ArgentinaSpace API")
 
 # Routers
-app.include_router(openaq.router, prefix="/openaq")
+app.include_router(tempo_router.router, prefix="/tempo")
 
 # CORS
 app.add_middleware(
     CORSMiddleware,
     allow_origins=[
         "http://localhost:5173",
         "http://127.0.0.1:5173",
         "http://localhost:4173",
         "http://127.0.0.1:4173",
     ],
     allow_credentials=True,
     allow_methods=["*"],
     allow_headers=["*"],
 )
 
+
 @app.get("/")
 def health():
     return {"status": "ok"}
